crates/bevy_ecs/src/world/mod.rs

```diff
@@ -788,6 +788,12 @@ impl World {
         unsafe { self.get_resource_with_id(component_id) }
     }
 
+    pub fn get_many_res<'w, MR: ManyRes<'w>>(&'w self) -> MR::Item {
+        MR::get_resources(self)
+    }
+    pub fn get_many_res_mut<'w, MR: ManyResMut<'w>>(&'w mut self) -> MR::Item {
```

Comment:
In addition to the above, this should also explain that you can only request a resource of a given type once, to avoid UB.